alot/commands/globals.py

```diff
@@ -414,9 +414,24 @@ def __init__(self, buffer=None, force=False, redraw=True, **kwargs):
 
     @inlineCallbacks
     def apply(self, ui):
+        def last_buffer_no_close():
+            """Helper that prints a message to the log and notifies if this is
+            the last buffer and the settings don't allow closebuffer to
+            exit."""
+            msg = ('not closing last remaining buffer as '
+                   'global.quit_on_last_bclose is set to False')
+            logging.info(msg)
+            ui.notify(msg, priority='error')
+
         if self.buffer is None:
             self.buffer = ui.current_buffer
 
+        # If there is only one buffer and the settings don't allow using
+        # closebuffer to exit, then just stop.
```

Comment:
The formulation of this docstirng is rather confusing:
actually, the settings spoecify not to exit the UI when/after closing the last buffer.

This also means that we *should* close the last buffer but keep the ui running (without open buffers), so we do not want to abort here..